.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,51 @@
+---
+name: 🐞 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
+  Or your issue may be closed.
+
+  👉🏽 Need help or tech support? Please don't open an issue!
+  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs
+
+  ❤️ Rollup? Please consider supporting our collective:
+  👉 https://opencollective.com/rollup/donate
+-->
+
+- Operating System:
+- Node Version:
+- NPM Version:
+- Rollup Version:
+- ${package} Version:
+
+### Expected Behavior
+
+
+### Actual Behavior
+
+
+### Code
+
+```js
+// rollup.config.js
+
+// If your Rollup config is meaty 🍖 and over 20 lines long, please provide a
```

Comment:
Rollup configs usually do not tend to get overly large but having a repo is nice in any case :)

Reply:
I'm open to reformatting this. I did see a bunch that were sizable when lots of plugins were added, so went with the better safe than sorry approach. If it's not something that happens frequently though, it probably doesn't have to be mentioned.